### DISCUSSION TITLE
Gradient Effect

### DISCUSSION THREAD

**Discussion by kmarler18** [tailwindlabs/tailwindcss#10162] [Help] [answered]
Hey Everyone,

I'm having trouble achieving the result in the photo directly below. Based on the Sketch file from my designer it appears there are three layers creating the gradient effect.

1. The gray "X" background that covers the entire header that implements background-color: #D8D8D8.
2. A red background that only covers half of the header that implements background: #E5403D and either mix-blend-mode: multiply or background-blend-mode: multiply (the Sketch file is ambiguous).
3. And another red background that only covers half of the header that implements background-image: linear-gradient(90deg, #D03E3D 6%, rgba(241,81,80,0.50) 100%).

![Screen Shot 2022-12-23 at 12 00 55 AM](https://user-images.githubusercontent.com/96091818/209274097-0565dca0-8012-4ed3-bfe8-31d7bd4a2e3a.png)

[Here's my code](https://drive.google.com/file/d/1Fmn8H-6KXbMF7kEucrPJiXZVleQ2XeuJ/view?usp=share_link), and my output is below. Notice how the gray "X" background doesn't have the fade effect as it moves toward the bottom of the header, unlike the photo above. 

![Screen Shot 2022-12-22 at 11 41 41 PM](https://user-images.githubusercontent.com/96091818/209272536-bc2f16e6-40a3-44ea-83bf-f96c84eb32bf.png)

I'm wondering if someone would be willing to check my code and give me a second pair of eyes to see where I'm going wrong.

Thanks a million,
Kyle

**Comment by wongjn** [comment, marked as answer]
You'd have to look at splitting the effect like the sketch file and having separate elements – since in your code, the `mix-blend-mode` on a parent affects the whole element – [here is the adjusted code in a Tailwind Play](https://play.tailwindcss.com/uCJwFi9kid).

**Reply by kmarler18** [reply]
@wongjn Legend. 

As always, thank you.

Now I've just got to get someone to figure out this [text-decoration issue](https://github.com/tailwindlabs/tailwindcss/discussions/10144).
